src/imgui/ImGuiDebugger.cc

```diff
@@ -629,13 +629,39 @@ void ImGuiDebugger::drawDisassembly(CPURegs& regs, MSXCPUInterface& cpuInterface
 								}
 							});
 
+							enum Priority {
+								LOWEST_PRIORITY = 0,
+								MEDIUM_PRIORITY,
+								HIGHEST_PRIORITY
+							};
+							Priority currentPriority = LOWEST_PRIORITY;
+							std::vector<std::string_view> bestCandidates;
+							auto add_candidate = [&](const Symbol* sym, enum Priority newPriority) {
```

Comment:
Very much a detail:
In C it's required to repeat `enum` before `Priority`, in C++ it's not (and usually it's left out).